Gemfile

```diff
@@ -5,3 +5,13 @@ git_source(:github) { |repo| "https://github.com/#{repo}.git" }
 
 # Specify your gem's dependencies in yabeda-activejob.gemspec.
 gemspec
+
+group :development, :test do
```

Comment:
Most of these are already in the `yabeda-activejob.gemspec` file. Would it make more sense to add the new dependencies there?

Reply:
Rubocop yelled at me to not do that so I moved it out of there.  Also we need these here to run them in CI.

Reply:
I read through some conversations and blog posts and I _think_ I understand the reasoning. I'll give this one last read and approve.